src/hotspot/cpu/arm/vm_version_arm_32.cpp

```diff
@@ -110,10 +110,14 @@ void VM_Version::early_initialize() {
   // use proper dmb instruction
   get_os_cpu_info();
 
+#if defined(__linux__)
```

Comment:
Should this be `#ifdef KUSER_HELPER_VERSION_ADDR` instead?  I'd prefer this to be in terms of functionality rather than OS if that makes sense.

Reply:
KUSER_HELPER_VERSION_ADDR is defined for every platform, I can ifdef it in https://github.com/battleblow/openjdk-jdk11u/blob/bsd-port/src/hotspot/cpu/arm/vm_version_arm.hpp#L92 and `ifdef KUSER_HELPER_VERSION_ADDR` here. It's really linux specific

Reply:
Makes sense.  Let's leave if as `#ifdef __linux__` then.